drivers/include/periph/dac.h

```diff
@@ -9,144 +10,117 @@
 /**
  * @defgroup    drivers_periph_dac DAC
  * @ingroup     drivers_periph
- * @brief       Low-level DAC peripheral driver
+ * @brief       DAC peripheral driver interface
+ *
+ * Similar to the ADC driver interface (@ref drivers_periph_adc), the DAC
+ * interface uses the concept of lines, corresponds to a tuple of a DAC device
+ * and a DAC output channel.
+ *
+ * The DAC interface expects data to be served as a 16-bit unsigned integer,
+ * independent of the actual resolution of the hardware device. It is up to the
+ * DAC driver, to scale the given value to the maximal width that can be
+ * handled. The device driver should, on the other hand, implement the DAC in a
+ * way, that it will use the bit width, that comes closest to 16-bit.
+ *
+ * This kind of 'auto-scaling' is quite sufficient for this interface, as
+ * standard DAC peripherals use a fixed conversion resolution internally anyway,
+ * so that any particular bit-width configuration on this driver level would not
+ * have much effect...
  *
  * @{
  * @file
- * @brief       Low-level DAC peripheral driver interface definitions
+ * @brief       DAC peripheral driver interface definition
  *
  * @author      Simon Brummer <simon.brummer@haw-hamburg.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef DAC_H
-#define DAC_H
+#ifndef PERIPH_DAC_H
+#define PERIPH_DAC_H
 
 #include <stdint.h>
+
+#include "periph_cpu.h"
 #include "periph_conf.h"
+/* TODO: remove once all platforms are ported to this interface */
+#include "periph/dev_enums.h"
```

Comment:
nope, you are correct, I'll remove this, as we don't want to merge new stuff anyway, that uses this...